Makefile

```diff
@@ -348,7 +348,7 @@ test-race:
 ################################################################################
 .PHONY: test-integration
 test-integration: test-deps
-		gotestsum \
+		CGO_ENABLED=1 gotestsum \
```

Comment:
Running `test-integration` requires CGo, so this enables it in case the shell's environment has `CGO_ENABLED=0` (like mine)

Reply:
It’s because it uses `-race`